packages/@ama-terasu/core/src/module/common-config.ts

```diff
@@ -20,6 +20,11 @@ export const yargsAmaCli = yargs(hideBin(process.argv))
     description: 'Display the log in verbose mode',
     type: 'boolean'
   })
+  .options('local-only', {
+    global: true,
+    description: 'Use installed package only',
```

Comment:
Looking at the implementation I'm not sure this is the case.

If you installed a remote module before, the next run with `--local-only` won't detect it because it looks only at the `cli`'s package and not the `dyn_modules` thing